doc/rados/operations/monitoring.rst

```diff
@@ -738,3 +738,37 @@ Print active connections and their TCP round trip time and retransmission counte
 
 	248     89      1       mgr.0   863     1677    0
 	3       86      2       mon.0   230     278     0
+
+Tracking Data Availability Score of a Cluster
+=============================================
+
+Ceph internally tracks the data availability of each pool in a cluster.
+To check the data availability score of each pool in a cluster, 
+the following command can be invoked: 
+
+
+.. prompt:: bash $
+
+   ceph osd pool availability-status
+
+Example output:  
+
+.. prompt:: bash $
+
+   POOL       	UPTIME  DOWNTIME  NUMFAILURES  MTBF  MTTR  SCORE 	AVAILABLE
+   rbd            	2m   	21s        	1	2m   21s  0.888889      	1
+   .mgr          	86s    	0s        	0	0s	0s     	1      	1
+   cephfs.a.meta 	77s    	0s        	0	0s	0s     	1      	1
+   cephfs.a.data 	76s    	0s        	0	0s	0s     	1      	1
+
+A pool is considered ``unavailable`` when at least one PG in the pool 
+becomes inactive or there is at least one unfound object in the pool. 
+Otherwise the pool is considered ``available``. 
+
+We first calculate the Mean Time Between Failures (MTBF) and 
+Mean Time To Recover (MTTR) from the uptime and downtime recorded 
+for each pool and arrive at the availability score 
+by finding ratio of MTBF to total time (ie MTTR + MTBF).  
+
+The score is updated every 5 seconds. This interval is currently 
```

Comment:
"the ratio"
s/5/five/

I'm not sure that "(ie MTTR + MTBF)" is what the first part of that sentence is conveying, my sense would be to truncate the sentence after "time".

Reply:
That makes sense, done!